src/components/sticky-object.js

```diff
@@ -48,7 +48,7 @@ AFRAME.registerComponent("sticky-object", {
     this.heldRightRemote = heldRightRemote;
 
     if (!almostEquals(0.001, this.prevScale, this.el.object3D.scale)) {
-      if ((this.heldLeftHand || this.heldRightHand || this.heldRightRemote) && !this.wasScaled) {
+      if ((!this.el.components.networked || NAF.utils.isMine(this.el)) && !this.wasScaled) {
```

Comment:
Rather than have each `sticky-object` check to see if it's owned and scaled, how about we set the appropriate attribute on the `ammo-body` at the same time that we begin scaling an (owned) object: https://github.com/mozilla/hubs/blob/master/src/systems/two-point-stretching-system.js#L39

Reply:
(And likewise set the appropriate attribute(s) when using the scaling button in the menu)